common/src/stack/switch/pylib/switch/m7800.py

```diff
@@ -10,6 +11,10 @@
 def info(message):
 	syslog.syslog(syslog.LOG_INFO, f'm7800.py: {message}')
 
+def remove_blank_lines(lines):
```

Comment:
This seems like it would be useful elsewhere, and might make a good addition to `stack.util` if you moved it there.